vm_automation/esxiVm.py

```diff
@@ -108,18 +110,17 @@ def connect(self):
         return retVal
 
     def logMsg(self, strMsg):

```

Comment:
Can we extract this method into a separate file? A simple import for the method for each implementation will make life so much simpler.

Reply:
We can, but it's a fairly inconsequential function, and we'd have to add the logfile to it if we removed it from the class.  Right now that log file is attached to the class.  I'd need to look through to see if there would be other concerns.

Reply:
Having a central log vs class(instance) defined logs seems more appropriate for a library, possibly putting an originating class/instance reference in the log line would allow quick separation of log lines later?  Can be tabled for now but I would like to see this moved to common code.